Formula/podman.rb

```diff
@@ -42,7 +42,9 @@ def install
       bin.install "bin/gvproxy"
     end
 
-    system "make", "install-podman-remote-#{os}-docs"
+    inreplace "Makefile", "install-podman-remote", "podman-remote", false
```

Comment:
This seems like the wrong thing to do. We should wait for upstream to change the `Makefile` instead of modifying it in the formula.